tpgtools/templates/sweeper.go.tmpl

```diff
@@ -45,9 +50,14 @@ func init() {
 	})
 }
 
+{{- if $.Undeletable }}
```

Comment:
W/o metrics scope, I think we can remove `Undeletetable`? I'd rather not keep around code that isn't in use.

Reply:
It will be used whenever we implement kms resources in tpgtools.

Reply:
We can add it back then, right? That's also likely a ways away, given they're implemented in MMv1 and aren't gonna be high-impact to move to DCL.

Reply:
Should be gone now after latest commit.